test/Azure.Functions.Cli.Tests/E2E/CreateFunctionTests.cs

```diff
@@ -71,6 +71,23 @@ public async Task create_template_function_using_alias()
             }, _output);
         }
 
+        [Fact]
+        public async Task create_function_custom()
+        {
+            await CliTester.Run(new RunConfiguration
+            {
+                Commands = new[]
+                {
+                    "init . --worker-runtime custom",
+                    "new --template \"Http Trigger\" --name testfunc"
+                },
+                OutputContains = new[]
+                {
+                    "The function \"testfunc\" was created successfully from the \"Http Trigger\" template."
```

Comment:
should this verify function.json content?

Reply:
I'm just following the pattern , I'll explore if there's a way to verify content once templates are released.